<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetOptions</class>
 <widget class="QWidget" name="ResetOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QWidget" name="settingsHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="settingLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reset options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>settingLabel</zorder>
   <zorder>back</zorder>
  </widget>
  <widget class="QPushButton" name="resetValues">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>280</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset values
and shutdown</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetConfig">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>280</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset configuration
and shutdown</string>
   </property>
  </widget>
  <widget class="mold::Confirmation" name="popUp" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>280</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mold::Confirmation</class>
   <extends>QWidget</extends>
   <header>gui/Confirmation.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
